app/controllers/manual_sections_controller.rb

```diff
@@ -17,10 +17,7 @@ def new
   end
 
   def create
-    section_data = params["section"]
-    section_data["manual_content_id"] = params["manual_content_id"]
-
-    @section = Section.new(section_data)
+    @section = Section.new(section_params.merge("manual_content_id" => params["manual_content_id"]))
```

Comment:
Prefer to use Ruby 1.9 hash syntax -> `manual_content_id: params["manual_content_id"]`